nautobot_ssot/jobs/examples.py

```diff
@@ -79,9 +87,6 @@ class LocationModel(NautobotModel):
     tenant__name: Optional[str]
     description: str
 
-    # Not in _attributes or _identifiers, hence not included in diff calculations
-    pk: Optional[UUID]
```

Comment:
Are you removing this for a particular reason?  I only ask because I know I've questioned how/where to use a non diff'd attribute in my models and having one in the example might be helpful?

Reply:
Oh I just saw your conversation with Leo.

Reply:
Agreed on that front. I think the documentation update that @Kircheneer is doing will illustrate that bit too.